### PR TITLE
BAU: Regression fixes

### PR DESCRIPTION
**Pull request by willfish**
### Jira link

BAU

### What?

I have added/removed/altered:

- [x] Adjusts the validDate to reflect changed start dates on various measures
- [x] Updates tests to reflect change to validDate and the current state of measures on this date

### Why?

I am doing this because:

- To make sure the build is green and that we have proper coverage

### Have you? (optional)

- [ ] Created clear commits for traceability?
- [ ] Avoided brittle tests which need constant fixing due to data/or static content changes?
- [ ] Made sure there isn't already a test for this feature?
- [ ] Avoided testing logic that isn't part of an integration between multiple systems and that is effectively a unit test in the application already?
- [ ] Reviewed changes with affected stakeholders?

### CODE REVIEW

File: cypress/e2e/DutyCalculator/2.RoW-GB/RoW-GB208--e2e.cy.js [hunk 1]
```diff
@@ -62,6 +63,5 @@ describe('| RoW-GB208--e2e.spec | Special calculations - alcohol % + sugar % |',
     cy.contains('£3,916.80');
     // tariff preference rate for Singapore
     cy.contains('Option 2: Tariff preference - Singapore');
-    cy.contains('0.00 GBP / 100 kg/net/%sacchar.');
```
Comment (willfish) [inline cypress/e2e/DutyCalculator/2.RoW-GB/RoW-GB208--e2e.cy.js:65]: We should probably validate the presentation of this. Do you have a tariff preference with Viet Nam that shows this measurement unit for this commodity?

Comment (bhavanibandla) [reply cypress/e2e/DutyCalculator/2.RoW-GB/RoW-GB208--e2e.cy.js:65]: Updated tariff preference with Viet Nam

File: cypress/e2e/DutyCalculator/2.RoW-GB/RoW-GB204-e2e.cy.js [hunk 2]
```diff
@@ -9,7 +9,7 @@ describe('| RoW-GB204-e2e.spec | 🇦🇫🇸Afghanistan to 🇬🇧 GB  |', fun
     cy.originList({value: 'Afghanistan'});
     cy.customsValue({monetary: '500.00', shipping: '100.00', cost: '250.00'});
     //  cy.quantity({tne:'1',kgm:'1',dap:'1'})
-    cy.additionalCode({uk: '2601'});
+    cy.additionalCode({uk: 'none'});
```
Comment (willfish) [inline cypress/e2e/DutyCalculator/2.RoW-GB/RoW-GB204-e2e.cy.js:12]: :raised_hands:

File: cypress/e2e/DutyCalculator/2.RoW-GB/RoW-GB204-e2e.cy.js [hunk 3]
```diff
@@ -21,27 +21,25 @@ describe('| RoW-GB204-e2e.spec | 🇦🇫🇸Afghanistan to 🇬🇧 GB  |', fun
...
     // Excise code
+    cy.contains('419 - Wine with an alcoholic strength exceeding 22% ABV');
     cy.exciseCode('419');
-
     cy.confirmPage();
     cy.dutyPage();
-
     cy.contains('Option 1: Third-country duty');
     cy.contains('Option 2: Tariff preference - GSP – Least Developed Countries');
-    cy.contains('419 - Wine with an alcoholic strength exceeding 22% ABV');
```
Comment (willfish) [inline cypress/e2e/DutyCalculator/2.RoW-GB/RoW-GB204-e2e.cy.js:44]: We should show this on the duty page. This is broken and I might have broken it - looking into this now.

Comment (willfish) [reply cypress/e2e/DutyCalculator/2.RoW-GB/RoW-GB204-e2e.cy.js:44]: This is what should be shown:

image

This is what is shown:

image

File: cypress/e2e/DutyCalculator/3.RoW-NI/RoW-NI304c-delta-e2e.cy.js [hunk 4]
```diff
@@ -36,8 +36,8 @@ describe('| Row-NI304c-delta.spec.js |turnover > £500,000 |  🔼 Delta Route -
     cy.contains('6307 90 92 00 (2600)');
 
     cy.contains('Option 1: Third-country duty');
-    cy.contains('Third-country duty (EU)');
-    cy.contains('EU import duties apply, as the difference between the UK third country duty and the EU third country duty exceeds 3% of the customs value of your trade.');
+    // cy.contains('Third-country duty (EU)');
```
Comment (willfish) [inline cypress/e2e/DutyCalculator/3.RoW-NI/RoW-NI304c-delta-e2e.cy.js:39]: This shouldn't have been missing - the second third country duty, here, should have shown as using an EU measure. Reinstating this after recent fixes

**Pull request merged**